.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- 👉👉👉 Provide a general summary of your changes. 👈👈👈 -->
+
+## Motivation and Context
+<!--- 💭💡💭 Why is this change required? What problem does it solve? 💭💡💭 -->
+<!--- 🔗 If it fixes an open issue, please link to the issue here. 🔗 -->
+
+## Description
+<!--- 📝🎯📝 Describe your changes in detail. 📝🎯📝 -->
+
+## How has this been tested?
+<!--- 🙆👌🙆 Please describe in detail how you tested your changes. 🙆👌🙆 -->
+<!--- Include details of your testing environment, details of your manually tests, snippet code or commands of your manually tests, table of experiments results metrics, tests ran to see how your change affects other areas of the code, etc. -->
```

Comment:
I am not so sure about this, is it supposed to be "manual tests" instead of "manually tests"?

Reply:
Thanks.